recipe/meta.yaml

```
@@ -1,44 +1,56 @@
 {% set name = "plyvel" %}
-{% set version = "1.5.0" %}
+{% set version = "1.5.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: cd918e0b31690abcd3d202a8742caf951ab2fe1573de7af71c38456847f9202b
+  sha256: 3caf6009e4fc24fbf8712d3f5ef3da7e56495c26a488df2161218fc34e46019c
 
 build:
   number: 0
-  skip: True  # [py<36]
-  script: {{ PYTHON }} -m pip install . -vv --no-deps
+  # 3.12 support introduced in 1.5.1: https://github.com/wbolster/plyvel/blob/main/NEWS.rst#plyvel-151
+  skip: True  # [py<36 or py>312]
+  script:
+    - cython --cplus --fast-fail --annotate plyvel/_plyvel.pyx -vv
+    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   build:
+    - {{ stdlib('c') }}
     - {{ compiler('cxx') }}
+    - cython
```

https://github.com/wbolster/plyvel/blob/1.5.1/requirements-dev.txt#L3
```suggestion
    - cython >=0.17
```